transformer_class.py

```diff
@@ -171,12 +72,7 @@ def add_parameters(self, parser):
         Arguments:
             parser: instance of argparse
         """
-        # pylint: disable=no-self-use
-        parser.add_argument('--logging', '-l', nargs='?', default=os.getenv("LOGGING"),
-                            help='file or url or logging configuration (default=None)')
-
-        parser.epilog = configuration.TRANSFORMER_NAME + ' version ' + configuration.TRANSFORMER_VERSION + \
-                        ' author ' + configuration.AUTHOR_NAME + ' ' + configuration.AUTHOR_EMAIL
+        Transformer.add_parameters(self, parser)
```

Comment:
If this is just a call through to the base class, then it doesn't need to be overridden (then this method could be removed)

Reply:
It was meant to be a call to agpypeline's Transformer class; just fixed